sample/basic-app.js

```diff
@@ -52,7 +75,7 @@ pipeline.source(Stream.consumer({ name: 'process-mapper' }))
     // throw new Error('Kaka punjabi');
     next(data, err);
   })
-  .flow(Batch.map({ number: 5, timeout: 30000, groupBy: ["to", "from"], attributes: ["num", "from"], type: "leveldb" },
+  .flow(Batch.map({ number: 5, timeout: 30000, groupBy: ["to", "from"], attributes: ["num", "from" , "_timestamp"], timestamp : "time" },
```

Comment:
user will not be aware of _timestamp.
you should programmatically return two parameters in the result (similar to our group by parameters)
1.  the field mention in as timestamp 
2. time of message Arrival